webui/src/views/AddPlanModal.tsx

```diff
@@ -560,6 +576,100 @@ const RetentionPolicyView = () => {
     }
   };
 
+  const hasBucketedKeepLastN = retention?.policyTimeBucketed?.keepLastN > 1;
+
+  // Calculates the average duration of the last N snapshots
+  const calculateBucketedLastNDuration = () => {
```

Comment:
Hmm, I'm not sure I follow the intent here -- is it to help the user estimate a good value for last n that will be long enough to keep all of their snapshots?

Reply:
Rereading the PR description I think this makes sense to me actually! It might help if the tooltip text better conveys what it's being computed from e.g.

```{n} snapshots with your schedule {cron} represents an expected retention duration of {lower bound} and {upper bound} days, but may vary with manual backups or if intermittently online```

or some such text.

Reply:
one more nit: not sure that providing a lower and an upperbound makes sense -- may be more intuitive to simply pick one. Most people are probably concerned with the worst case minimum retention? Upper bound is always potentially unbounded e.g. if it's a laptop that gets unplugged for half-a-year before the next backup